2016/ap/problems/hello/hello.adoc

```diff
@@ -140,44 +138,44 @@ Go ahead and type `hello` (or the ever-popular `asdf`) on line 1 of the document
 
 Okay, with `hello.txt` still open in your workspace, notice that beneath your document is that terminal window. Notice that the window's prompt is, assuming you haven't otherwise manipulated it
 
-[source,bash]
+[source]
 ----
-~/workspace $
+~/workspace/ $
 ----
 
-(where `~/workspace` &ndash; i.e., inside a folder called `workspace` which is itself inside your home directory, the shorthand for which is `~` &ndash; is the directory you are currently in.) If that's the case, based on the file tree should be a `chapter1` directory somewhere inside, since we created it just a few moments ago. Let's confirm as much.
+(where `~/workspace/` &ndash; i.e., inside a folder called `workspace` which is itself inside your home directory, the shorthand for which is `~` &ndash; is the directory you are currently in.) If that's the case, based on the file tree should be a `chapter1` directory somewhere inside, since we created it just a few moments ago. Let's confirm as much.
```

Comment:
Let's not use HTML in AsciiDoc. Breaks rendering into other formats potentially (e.g., PDF). Just type an actual m-dash, —.

This should indeed be an m-dash, not n-dash, with no space on either side. If that breaks the backticks, change it to double backticks then the m-dash.